### ISSUE TITLE
MenuBar/TieredMenu: Incorrect ARIA Roles

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#5804]
## 1) Environment
- PrimeFaces version: ALL
- Does it work on the newest released PrimeFaces version? NO
- Does it work on the newest sources in GitHub? NO
- Application server + version: ALL
- Affected browsers: ALL

## 2) Expected behavior

MenuBar, MegaMenu, and TieredMenu should have proper ARIA roles for screenreaders.

```xml
<div>
    <ul role="menubar">
         <li role="none">
               <a href="#" role="menuitem">
         </li>
     </ul>
</div>
```

## 3) Actual behavior

These menus currently have this for ARIA which is incorrect according to https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html

```xml
<div role="menubar">
    <ul >
         <li role="menuitem">
               <a href="#">
         </li>
     </ul>
</div>
```

Which causes Google Lighthouse to report:
![image](https://user-images.githubusercontent.com/4399574/79562499-395cc280-8079-11ea-9439-dee98a30c7fd.png)

## 4) Steps to reproduce

Showcase: http://localhost:8080/showcase/ui/menu/menubar.xhtml

Run Google Lighthouse Accessibility audit. CC @MarkusChur

**Comment by MarkusChur** [comment]
TieredMenu has proper ARIA roles now.
